types/jsdom/jsdom-tests.ts

```diff
@@ -16,3 +16,6 @@ domWindow.BigUint64Array; // $ExpectType BigUint64ArrayConstructor
 domWindow.SharedArrayBuffer; // $ExpectType SharedArrayBufferConstructor
 domWindow.WebAssembly; // $ExpectType typeof WebAssembly
 domWindow.external; // $ExpectType External
+
+const resourceLoader = new jsdom.ResourceLoader();
+resourceLoader.fetch('https://example.com', {})!.abort; // $ExpectType () => void
```

Comment:
This should be moved to `types/jsdom/test/core.ts`, so that it’s tested for every **TypeScript** version, also it should probably use:
```ts
function testResourceLoader(resourceLoader: ResourceLoader) {
	// $ExpectType AbortablePromise<Buffer> | null
	resourceLoader.fetch('https://example.com', {});
}
```

Reply:
Yes, fixed. Thank you :smiley: